.travis.yml

```diff
@@ -30,3 +31,6 @@ script:
   - go vet ./management/...
   - test -z "$(golint ./Gododir/... | tee /dev/stderr)"
   - go vet ./Gododir/...
+  - set -e; for v in ./arm/... ./datalake-store/... ./Gododir/... ./management/... ./storage/...; do errcheck -ignoretests "$v"; done
```

Comment:
Travis fails with `errcheck`, take a look at [this build](https://travis-ci.org/Azure/azure-sdk-for-go/builds/182463249#L1570). And from errcheck [documentation](https://github.com/kisielk/errcheck/blob/master/README.md#exit-codes).